### PR TITLE
Fix warnings and reenable opendap2 test

### PR DESCRIPTION
**Pull request by fulminemizzega**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR also adds me as a maintainer.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -72,3 +73,4 @@ extra:
     - jhamman
     - david-ian-brown
     - khallock
+    - fulminemizzega
```
Comment (hmaarrfk) [inline recipe/meta.yaml:76]: Nice!

Comment (hmaarrfk) [reply recipe/meta.yaml:76]: We should rerender for this to take effect!

Comment (fulminemizzega) [reply recipe/meta.yaml:76]: My bad... this thing is too much automated

**Pull request merged**